content/soarian/overview.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Overview
+layout: soarian/overview
+---
+# Overview
+
+This document describes the resources that comprise Cerner’s Soarian Clinicals<sup>®</sup> implementation of the Health Level Seven<sup>®</sup> (HL7)<sup>®</sup> International Fast Healthcare Interoperability Resources<sup>®</sup> (FHIR)<sup>®</sup> standard. If you have any problems or requests, post them to our [developer group](https://groups.google.com/forum/#!forum/cerner-fhir-developers).
+
+* TOC
+{:toc}
+
+## Supported Versions
+
+Cerner’s Soarian Clinicals<sup>®</sup> implementation currently supports both the R4 First Normative Content (4.0.1) version and DSTU 2 Final (1.0.2) version of the HL7<sup>®</sup> FHIR<sup>®</sup> standard. Cerner’s implementation of the R4 version is ongoing and new resources and actions will be added over time.
```

Comment:
DSTU2 should be a single word, I believe.

Reply:
DSTU 2 is not a single word in FHIR site so as we discussed going with the same.

Reply:
That's fine. Thanks.